package.json

```diff
@@ -24,5 +24,6 @@
   },
   "files": [
     "id.js"
```

Comment:
Docs say README is always included by default https://docs.npmjs.com/files/package.json#files

I wonder if we should add `index.js` here though.